hedera-node/hedera-network-admin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/FreezeServiceImpl.java

```diff
@@ -55,9 +55,13 @@ public Set<StateDefinition> statesToCreate() {
             public void migrate(@NonNull final MigrationContext ctx) {
                 // Reset the upgrade file hash to empty
                 // It should always be empty at genesis or after an upgrade, to indicate that no upgrade is in progress
-                // Nothing in state can ever be null, so use Bytes.EMPTY to indicate an empty hash
+                // Nothing in state can ever be null, so use Type.DEFAULT to indicate an empty hash
                 final var upgradeFileHashKeyState = ctx.newStates().<ProtoBytes>getSingleton(UPGRADE_FILE_HASH_KEY);
                 upgradeFileHashKeyState.put(ProtoBytes.DEFAULT);
+                final var freezeTimeKeyState = ctx.newStates().<Timestamp>getSingleton(FREEZE_TIME_KEY);
+                freezeTimeKeyState.put(Timestamp.DEFAULT);
+                final var lastFrozenTimeKeyState = ctx.newStates().<Timestamp>getSingleton(LAST_FROZEN_TIME_KEY);
```

Comment:
Are we sure this should be reset?  I may be way off, but it seems to me that last frozen should tell the system the freeze time of the last upgrade or restart...

Reply:
I think the idea here is that it needs to be non-null, so by using the default Timestamp, it is basically saying the last freeze was 40+ years ago.